src/Raven.Server/Smuggler/Documents/DatabaseDestination.cs

```diff
@@ -137,12 +145,34 @@ public DatabaseDocumentActions(DocumentDatabase database, BuildVersionType build
 
             public void WriteDocument(Document document)
             {
-                _command.Add(document);
+                _command.Add(new DocumentItem
+                {
+                    Type = DocumentType.Document,
+                    Document = document,
+                });
+
+                HandleBatchOfDocumentsIfNecessary();
+            }
+
+            public void WriteAttachment(StreamSource.AttachmentStream attachment)
```

Comment:
This won't be an issue, but I removed the HandleBatchOfDocumentsIfNecessary from attachments.